<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- sub-workflow workflow autogenerated by Arbiter on 2018-03-14 13:37:34 -->
<workflow-app xmlns="uri:oozie:workflow:0.2" name="main-workflow">
    <start to="generateSubWorkflow"/>
    <action name="create_table2">
        <shell xmlns="uri:oozie:shell-action:0.1">
            <job-tracker>${jobTracker}</job-tracker>
            <name-node>${nameNode}</name-node>
            <configuration>
                <property>
                    <name>testProperty</name>
                    <value>testProperty</value>
                </property>
            </configuration>
            <exec>buildSubWorkflow.sh</exec>
        </shell>
        <ok to="runSubWorkflow"/>
        <error to="kill"/>
    </action>
    <action name="runSubWorkflow">
        <ok to="end"/>
        <error to="kill"/>
    </action>
    <kill name="kill">
        <message>Workflow sub-workflow has failed with msg: [${wf:errorMessage(wf:lastErrorNode())}]</message>
    </kill>
    <end name="end"/>
</workflow-app>